### DISCUSSION TITLE
Kayıkla karşıya geçme sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#6] [Zeka Sorusu]
İstanbul Boğazı'ndan 4 kişi kayık ile karşı yakaya geçmek istemektedir. Kayık en fazla iki kişi taşımakta ve kayığı bir kişi kullanmaktadır. Her seferinde kayığı kullanan kişi değişmektedir. Kayığı, karşı yakaya Ayfer 50 dakikada, Serkan 40 dakikada, Bayram 20 dakikada ve Döndü 10 dakikada geçirebilmektedir. Buna göre bu dört kişi en az kaç dakikada karşı yakaya geçebilir?

**Comment by FatihOnder81** [comment]
bayram la döndü beraber gider ve döndü geri döner  20+10=30 ayfer la serkan beraber gider ve bayram geri döner 50+40=90 bayram la döndü beraber gider 20 toplam 130 olur hocam

**Reply by sonsuzus** [reply]
Soruyu dikkatli oku.

**Comment by Furkanyildirm** [comment]
Bayram ve Döndü Bayram'ın öncülüğünde gider (20)
Döndü geri döner (10)
Serkan ve Ayfer Serkan'ın öncülüğünde gider (40)
Bayram kayığı devralır ve geri döner (20)
Döndü ile Bayram Döndü 'nün öncülüğünde gider (10)
yani hepsi en erken 20+10+40+20+10=100 dakikada karşıya geçerler
